src/app/api/quality_matrix/quality_matrix.py

```diff
@@ -138,6 +138,10 @@ async def items_in_response(response: Response) -> dict:
     return response.aggregations.to_dict().items()
 
 
+REPLICATION_SOURCE_ID = "ccm:replicationsource"
```

Comment:
I want to refactor all these XYZAttributes (elasticsearch, collection, learningmaterials) into one soon. This would be part of it, too, yes. Is it alright if I push it to another PR?